src/plugins/console/server/lib/spec_definitions/json/generated/snapshot.clone.json

```diff
@@ -9,6 +9,6 @@
     "patterns": [
       "_snapshot/{repository}/{snapshot}/_clone/{target_snapshot}"
     ],
-    "documentation": "https://opensearch.org/docs/latest/opensearch/rest-api/"
+    "documentation": "https://opensearch.org/docs/"
```

Comment:
I agree that it is the closest docs page we currently have, but I hesitated to include this link because there was no `_clone` pattern mentioned there. If that is fine I will do as suggested.

Reply:
Yeah - we'll need to do a separate audit to bring better alignment between the methods supported in the console and what's actually documented on the OpenSearch side.